runtimes/node-17.0/server.js

```diff
@@ -17,9 +17,14 @@ const server = micro(async (req, res) => {
         payload: body.payload ?? '',
     };
 
+    console.stdlog = console.log.bind(console);
```

Comment:
can this be some our variable? Sounds scarry to set variable on `console` object

Reply:
it can be, will update to that

Reply:
Let's keep it for internal use